config.xml

```diff
@@ -60,6 +60,8 @@
     <preference name="AutoHideSplashScreen" value="false" />
     <preference name="phonegap-version" value="cli-7.1.0" />
     <preference name="HeaderColor" value="#eb7125" />
+    <engine name="android" spec="^6.4.0" />
+    <engine name="ios" spec="^4.5.4" />
```

Comment:
Amazing change 👍

Reply:
I know, it's the best thing with the entire PR :D